forecasting_sa/models/abstract_model.py

```diff
@@ -127,18 +127,22 @@ def calculate_metrics(
         print("starting calculate_metrics")
         to_pred_df = val_df.copy()
         to_pred_df[self.params["target"]] = np.nan
-        to_pred_df = pd.concat([hist_df, to_pred_df])
+        to_pred_df = pd.concat([hist_df, to_pred_df]).reset_index(drop=True)
```

Comment:
`hist_df` and `to_pred_df` have different data type for their indices (datetime and int) and `PandasDataset.from_long_dataframe()` method fails. So, I'm dropping the index since it's not used.